### DISCUSSION TITLE
U8g2 HAL for Kendryte SDK Platform. (K210 Chipset)

### DISCUSSION THREAD

**Discussion by sreedevk** [olikraus/u8g2#1440] [Show and tell]
Hey All, I've created this very fast HAL (using DMAC) for the Kendryte SDK Platform. (K210 Chipset - Boards like MAixBit & MAixduino. I've only implemented i2c_hw for now. 
Contributors are welcome. I hope people find it useful. 
K210 is pretty powerful chip and using DMAC, the refresh rates & transmission speeds are crazy.

https://github.com/sreedevk/u8g2-kendryte-sdk-hal

**Comment by olikraus** [comment]
Thanks a lot. Nice code and good documentation.
